src/scancode/cli.py

```diff
@@ -561,7 +562,7 @@ def _scanit(paths, scanners, scans_cache_class, diag, timeout=DEFAULT_TIMEOUT):
             if scan_result.get('scan_errors'):
                 success = False
 
-    return success, rel_path
+    return success, rel_path, scan_result
```

Comment:
I feared something like that :-/ I'd be grateful for a proposal how to do it properly, esp. since you told me that I shouldn't access the cache directly to get this kind of information, IIRC :-D